collect_app/src/test/java/org/odk/collect/android/mainmenu/MainMenuButtonTest.kt

```diff
@@ -0,0 +1,41 @@
+package org.odk.collect.android.mainmenu
+
+import android.app.Application
+import android.widget.TextView
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.hamcrest.MatcherAssert.assertThat
+import org.hamcrest.Matchers.equalTo
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.odk.collect.android.R
+
+@RunWith(AndroidJUnit4::class)
+class MainMenuButtonTest {
+    private val context: Application =
+        ApplicationProvider.getApplicationContext<Application>().also {
+            it.setTheme(R.style.Theme_Material3_Light)
+        }
+
+    private lateinit var mainMenuButton: MainMenuButton
```

Comment:
You should just be able to assign this here without `lateinit`.

Reply:
Done.